common/src/main/java/com/microsoft/identity/common/internal/request/MsalBrokerRequestAdapter.java

```diff
@@ -105,11 +106,28 @@
     public BrokerRequest brokerRequestFromAcquireTokenParameters(@NonNull final InteractiveTokenCommandParameters parameters) {
```

Comment:
You might want to double check if these values are used everywhere.

For one, constructTokenEndpointForAcquiringPRT() is not using them.

Reply:
AFAIK, they are NOT used in most places...that's why we had developed the command line mechanism...those ones are injected everywhere.

Reply:
They're not, but that's only because these request construction mechanisms don't use common.  That command line mechanism doesn't work for people who want to use this feature without building it, does it?  FWIW, I was able to recommend an alternate path - just pass the query parameters, and that works without this change.